src/Shared/Microsoft.DotNet.Kusto/SharedKustoQueries.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Data;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Kusto
+{
+    public static class SharedKustoQueries
+    {
+        public static MultiProjectKustoQuery CreateBuildTimesQueries(string repository, string branch, int days)
```

Comment:
>MultiProjectKustoQuery [](start = 22, length = 22)

Is "MultiProjectKustoQuery" already in this DLL? If so, that's weird... it looks some business logic slipped into this DLL when I wasn't looking. :-(

Reply:
Yeah that came in with my last big PR. You wanted it to be explicit, this is the way I went. Since I'm messing with the code, we can talk about a better way to do this.